### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: test/v1/driver.test.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ describe('driver', () => {
 
   it('should handle connection errors', done => {
     // Given
-    driver = neo4j.driver("bolt://localhoste", sharedNeo4j.authToken);
+    driver = neo4j.driver("bolt://localhost", sharedNeo4j.authToken);
```
Comment (lutovich) [inline test/v1/driver.test.js]: @jsoref this spelling error is intentional. Test verifies connection errors. Build is red because this test fails.

Comment (jsoref) [reply test/v1/driver.test.js]: Would "local-host" suffice?

Comment (lutovich) [reply test/v1/driver.test.js]: I think yes

**Comment by lutovich** [comment]
Hi @jsoref,

Could you please fix the "localhoste" test and rebase the PR?

**Comment by jsoref** [comment]
Will do.

**Comment by jsoref** [comment]
@lutovich: done

**Comment by lutovich** [comment]
Thank you @jsoref!

**Pull request merged**